server/graphql/v2/interface/Transaction.js

```diff
@@ -273,5 +285,27 @@ export const TransactionFields = () => {
           : null;
       },
     },
+    isRejected: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      async resolve(transaction, _, req) {
+        if (transaction.OrderId) {
+          const order = await req.loaders.Order.byId.load(transaction.OrderId);
+          return order.status === orderStatus.REJECTED;
+        } else {
+          return false;
+        }
+      },
+    },
+    refundTransaction: {
+      type: Transaction,
+      resolve(transaction) {
+        // If it's a sequelize model transaction, it means it has the method getRefundTransaction
+        // otherwise we just null
+        if (transaction && transaction.getRefundTransaction) {
```

Comment:
I'm not sure how can we end up with a `null` transaction here, or why we would end up with something else than a sequelize transaction?

Reply:
This was marked as resolved but there's no change, you may have forgotten to push your changes